packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -9015,6 +9015,34 @@ export default{
 			expect(std.err).toMatchInlineSnapshot(`""`);
 		});
 	});
+
+	describe("--dispatch-namespace", () => {
```

Comment:
can we please add one extra test in https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/__tests__/configuration.test.ts to verify we perform the correct `name` validation if arg.dispatchNamespace is present?

Reply:
we could test this by passing in a `name` value that is a `string` but **not** a valid Worker name. If `args["dispatch-namespace"]` is not undefined and a string, `name` validation should pass

Reply:
Added